doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -622,71 +606,3 @@ The Platform is assumed to provide configurable alerting and notification capabi
 | sec.std.011 | Standards | The Cloud Operator **should** conform to the ISO/IEC 27031 standard for business continuity; ISO/IEC 27031 - ISO/IEC 27031 is the international Standard for ICT readiness for business continuity | |
 
 <p align="center"><b>Table 2-21:</b> Security Recommendations</p>
-
-<!--
-**Backlog of Req**
-
-1. Manage discovery of resources, resource capabilities/features
```

Comment:
Please do not delete "Discovery of resources, ..."

Reply:
Should we uncomment it? Is there a plan to reintroduce this information in the published content?

Reply:
@EditKoselak No -- we want to work on it in the near future

Reply:
Can you please reintroduce the comment:
<!--
 Manage discovery of resources, resource capabilities/features
-->